### DISCUSSION TITLE
Embedded Playground Static Files

### DISCUSSION THREAD

**Discussion by S4nfs** [openfga/community#270] [General]
Where can I find the source code for the OpenFGA embedded playground that is built in Next.js?
```
<iframe title="Embedded Playground" src="https://play.fga.dev/sandbox/?fga_api_host={{.HTTPServerURL}}&fga_api_scheme=http&fga_api_token={{.PlaygroundAPIToken}}" frameborder="0" marginheight="0" marginwidth="0" width="100%" height="100%" scrolling="auto"></iframe>
```
 Is it open source? Cause, I don't want to integrate it into my organization's authentication system with an iframe pointing to a 3rd party, in case they decide to make their project closed source or introduce a paywall to lock in customers later.

**Comment by aaguiarz** [comment]
Hi @S4nfs 

The playground is not open source, and it's main objective is helping you to learn the product, not to be deployed in your production app. Actually, you can only embed it if you run OpenFGA in localhost.

We expect developers to operate the product in production using the [CLI](https://github.com/openfga/cli), editing models using the [VS Code Extension](https://marketplace.visualstudio.com/items?itemName=openfga.openfga-vscode), and iterating+testing their models using our [testing features](https://openfga.dev/docs/modeling/testing).

What is the problem you want to solve with the Playground?

**Reply by S4nfs** [reply]
Nothing to solve, i just found it easier to get the bird's-eye-view of the authorization models as i am new to OpenFGA and thought this would help me and my teammates better understand the model as it grows. CLI works fine. Thanks @aaguiarz.
